corehq/apps/app_manager/suite_xml.py

```diff
@@ -1449,6 +1455,16 @@ def entries(self):
                 }[form.form_type]
                 config_entry(module, e, form)
 
+                if self.app.commtrack_enabled:
+                    from .models import AUTO_SELECT_LOCATION
+                    datum, assertions = self.get_location_assertions(
+                        'commtrack-supply-point',
+                        'supply_point_id',
+                        AUTO_SELECT_LOCATION,
+                    )
+                    e.datums.append(datum)
+                    e.assertions.extend(assertions)
```

Comment:
See if `"instance('commcaresession')/session/data/supply_point_id" in form.source()`